addons/mail/static/src/messaging/service/messaging_service/messaging_service.js

```diff
@@ -39,19 +32,20 @@ const MessagingService = AbstractService.extend({
             do_action: (...args) => this.do_action(...args),
             do_notify: (...args) => this.do_notify(...args),
             do_warn: (...args) => this.do_warn(...args),
+            entities: undefined,
             isMessagingInitialized() {
-                if (!self.Entities) {
+                if (!this.entities) {
                     return false;
                 }
-                const messaging = self.env.entities.Messaging.instance;
-                if (!messaging) {
+                if (!this.messaging) {
                     return false;
                 }
-                return messaging.isInitialized;
+                return this.messaging.isInitialized;
             },
             messagingBus: new EventBus(),
             rpc: (...args) => this._rpc(...args),
-            trigger_up: (...args) => this.trigger_up(...args)
+            trigger_up: (...args) => this.trigger_up(...args),
+            windowNotification: window.Notification,
```

Comment:
maybe it's best nested object, so it's accessible with `this.env.window.Notification`

Reply:
refresh PR and check again